src/main/java/seedu/address/commons/core/Config.java

```diff
@@ -10,11 +10,13 @@
  */
 public class Config {
 
-    public static final Path DEFAULT_CONFIG_FILE = Paths.get("config.json");
+    public static final Path DEFAULT_CONFIG_FILE = Paths.get("config", "footer", "config.json");
 
+    public static final Path USER_PREFS_FILEPATH = Paths.get("config", "footer", "preferences.json");
     // Config values customizable through config file
     private Level logLevel = Level.INFO;
-    private Path userPrefsFilePath = Paths.get("preferences.json");
+
+    private Path userPrefsFilePath = Paths.get("config/footer/preferences.json");
```

Comment:
This seems to have caused some problem regarding the back slash and forward slash. Bur still thanks for changing the footer in the configration.